Makefile

```diff
@@ -110,18 +114,9 @@ clean:
 	rm -rf $(BINDATA) $(OUT_DIR)
 
 local-image:
-ifdef USE_BUILDAH
-	buildah bud $(BUILDAH_OPTS) -t $(IMAGE_TAG) -f $(DOCKERFILE) .
-else
-	sudo docker build -t $(IMAGE_TAG) -f $(DOCKERFILE) .
-endif
+	$(IMAGE_BUILD_CMD) build -t $(IMAGE) -f $(DOCKERFILE) .
```

Comment:
How do I pass additional options to the build now that `$(BUILDAH_OPTS)` are removed?

Reply:
we could add a var `IMAGE_BUILD_EXTRA_OPTS ?=`  
then it will look something like 
```
$(IMAGE_BUILD_CMD) build -t $(IMAGE) $(IMAGE_BUILD_EXTRA_OPTS) -f $(DOCKERFILE) .
```

and the make can be called

```bash
IMAGE_BUILD_EXTRA_OPTS=some extra args
make build
```
wdyt?

Reply:
The options probably need to go right after "build".  Saying that,
`IMAGE_BUILD_CMD=podman build`  why do we have another "build" on the line?  Same below for push.  Did you test this PR?

Reply:
@ArangoGutierrez I believe this wasn't addressed yet.  Again...  Have you tested this PR?  Also, the extra options need to be passed to the push too.